tests/vmi_networking_test.go

```diff
@@ -789,6 +790,104 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			table.Entry("without a specific port number", []v1.Port{}, defaultCIDR),
 			table.Entry("with custom CIDR", []v1.Port{}, customCIDR),
 		)
+
+		When("performing migration", func() {
```

Comment:
It is under "VirtualMachineInstance with masquerade binding mechanism", and this closure is indeed using such a binding for the test.

Reply:
It does not use the var defined for the context. I close my eyes since this context is already spoiled by the DescribeTable. Once we start refactoring of this, we have to chose Contexts wiser.